Makefile

```
@@ -1,29 +1,39 @@
-sources = src tests docs/.hooks/
+.DEFAULT_GOAL := all
 
 .PHONY: .uv  ## Check that uv is installed
 .uv:
 	@uv -V || echo 'Please install uv: https://docs.astral.sh/uv/getting-started/installation/'
 
+.PHONY: install
+install: .uv  ## Install package and dependencies for local development
+	uv sync --frozen --all-groups
+	pre-commit install --install-hooks
```

```suggestion
	uv run pre-commit install --install-hooks
```

And don't we need to add `pre-commit` to `pyproject.toml`?

---

yup, done.